mlflow/pytorch/_pytorch_autolog.py

```diff
@@ -0,0 +1,208 @@
+import gorilla
+import logging
+import mlflow.pytorch
+import os
+import pytorch_lightning as pl
+import shutil
+import tempfile
+from pytorch_lightning.core.memory import ModelSummary
+from pytorch_lightning.utilities import rank_zero_only
+from mlflow.utils.autologging_utils import try_mlflow_log, wrap_patch
+
+logging.basicConfig(level=logging.ERROR)
+
+every_n_iter = 1
+
+
+# autolog module uses `try_mlflow_log` - mlflow utility to log param/metrics/artifacts into mlflow
+# Eventhough the same can be achieved using
```

Comment:
@shrinath-suresh thanks for documenting this - should we file a feature request against https://github.com/PyTorchLightning/pytorch-lightning/issues / consider whether we can contribute the improvements in your callback back to Pytorch Lightning's MlflowLogger? It seems long-term it'd be ideal if Pytorch Lightning's MlflowLogger could be used for autologging

Reply:
And then I think we could replace this comment with something like

```suggestion
# TODO: Replace __MlflowPLCallback with Pytorch Lightning's built-in MlflowLogger once the issue in <link-to-issue> have been addressed
```